### ISSUE TITLE
5 posts (including CPTs)	5/5	 100%  0 media items

### ISSUE THREAD

**Issue by bnovak99** [humanmade/WordPress-Importer#173]
Just did a test with five posts that had images and the media files were not imported.

**Comment by tomjn** [comment]
@bnovak99 keeping in mind the repo hasn't been touched since 2017 ( 5 years ago ), I don't think there's enough information here to diagnose a problem, you'd have to describe where you were importing from, and importing to, if the URLs are both reachable, or the same, if the target server can read the source server, if they're both online or just one, etc

People regularly run into problems using the regular WP importer from not checking the media import box, or importing a WXR on the target server when the source has gone away already or has the same URL but on a different host. Be specific and detailed.

**Comment by tomjn** [comment]
Additionally, when you say the media files were not imported, can you be more specific? They were imported but lead to 404's? Images show in the post but they aren't in the media library? Posts have broken images and nothing in the media library? They show in the media library but not in the posts? They're on the disk but not in WordPress?

**Comment by tomjn** [comment]
see https://github.com/humanmade/WordPress-Importer/issues/171#issuecomment-663465056

**Comment by bnovak99** [comment]
I exported blog posts from one site Tools > Export > Posts > Category > Blog > Sept 2022

Then tried to import the xml file into another site using your WP Importer checking the box "Download and import file attachments".  Five posts were imported but no media files. No files in the media library. The posts are still pointing to the old site urls for the media.

**Comment by iamkingsleyf** [comment]
> I exported blog posts from one site Tools > Export > Posts > Category > Blog > Sept 2022
> 
> Then tried to import the xml file into another site using your WP Importer checking the box "Download and import file attachments". Five posts were imported but no media files. No files in the media library. The posts are still pointing to the old site urls for the media.

This one stopped working, as you can see they stopped updating it here but they do however update it for their internal use maybe try the official import plugin

**Comment by bnovak99** [comment]
Yeah I just tried the regular WordPress importer and then used the **Auto Upload Images** plugin for media files. Did the job perfectly.
